dipy/testing/decorators.py

```diff
@@ -52,9 +53,15 @@ def doctest_skip_parser(func):
 # xvfbwrapper) conditioned on an environment variable (that we set in
 # .travis.yml for these cases):
 use_xvfb = os.environ.get('TEST_WITH_XVFB', False)
+if use_xvfb == 'true':
```

Comment:
Hi @arokem. Questions :) I need some clarifications here. Does that mean that I have to write 'true' in the command line too? What we were using before? TEST_WITH_XVFB=1 right? Or am I missing something?